modules/installing-odh-components.adoc

```diff
@@ -35,6 +35,10 @@ endif::[]
 | Prometheus Operator
 | Community
 
+| authorino
```

Comment:
At the moment, authorino is not a component. Instead, is a prerequisite for KServe. So, I think that rather than adding a new row to the table, it should be added to the KServe row.

Reply:
Thanks, fixed.